### ISSUE TITLE
Cant inject

### ISSUE THREAD

**Issue by kkrap** [NMan1/Modern-Warfare-Warzone-Cheat#2]
I can inject with my dx12 base but not with this one. Do you know what could be the issue?

**Comment by NMan1** [comment]
Yeah i had some injecting issues as well when trying to manual map it. Try using a native inject. Probably something with the imports not be resolved or something idk

**Comment by kevstr** [comment]
This doesn't work with any injection method
Base source injects fine

**Comment by NMan1** [comment]
Your blatantly wrong I’ve used many injectors that work fine if you don’t manual map.
